mvp-spec/ad-server-implementation.md

```diff
@@ -0,0 +1,785 @@
+                "value": "7435313e-caee-4889-8ad7-0acd0114ae3c",
+                "source": {
+                    "domain": "operotor0.com",
+                    "timestamp": 1639589531,
+                    "signature": "12345_signature"
+                }
+            }
+        ],
+        "preferences": {
+            "version": 0,
+            "data": { 
+                "opt_in": true 
+            },
+            "source": {
+                "domain": "cmp1.com",
+                "timestamp": 1639589531,
+                "signature": "12345_signature"
+            }
+        }
+    },
+    "seed": {
+        "version": 0,
+        "transaction_id": 1234567,
+        "publisher": "publisher.com",
+        "source": {
+          "domain": "operator0.com",
+          "timestamp": 1639589531,
+          "signature": "12345_signature"
+        }
+    },
+    "transmissions": [
+        {
+            "version": 0,
+            "receiver": "ssp1.com",
+            "status": "success",
+            "details": "",
+            "source": {
+                "domain": "ssp1.com",
+                "timestamp": 1639589531,
+                "signature": "12345_signature"
+            }
+        },
+        {
+            "version": 0,
+            "receiver": "ssp2.com",
+            "status": "success",
+            "details": "",
+            "source": {
+                "domain": "ssp2.com",
+                "timestamp": 1639589531,
+                "signature": "12345_signature"
+            }
+        },
+        {
+            "version": 0,
+            "receiver": "dsp.com",
+            "status": "success",
+            "details": "",
+            "source": {
+                "domain": "dps.com",
+                "timestamp": 1639589531,
+                "signature": "12345_signature"
+            }
+        }
+    ]
+}
+```
+<!--partial-end-->
+
+### Step 7: Display the Addressable Content and make the Audit Log available
+
+Finally, the Addressable Content can be displayed and depending on the 
+implementation choice, the Audit Log can be added in a hidden tag in the
+DOM or log in to the console.
+
+### Transmissions with OpenRTB
+
+If the used protocol for offering the inventory is OpenRTB, the Ad Server needs
+to respect the following for integrating Prebid SSO.
+
+#### The OpenRTB Bid Request
+
+In step **Step 4**, the Ad Server must share the Prebid SSO Data in the 
+extensions of the Bid Request:
+
+First, The Transmission Request object in an OpenRTB request keeps the same structure.
+It is embedded in the `ext` field of each impression. It is 
+reachable at `imp`.`ext`.`prebid_sso`.
+
+Second, the Pseudonymous-Identifiers structure change in the OpenRTB request to take the
+advantage of the [Extended Identifiers](https://github.com/InteractiveAdvertisingBureau/openrtb/blob/master/extensions/2.x_official_extensions/eids.md). 
+One `eid` per Pseudonymous-Identifier. It is reachable at `user`.`ext`.`eids`
```

Comment:
Add line break before "comparting" (which should be "comparing"?)

Reply:
Done